pkg/policyfilter/state.go

```diff
@@ -130,8 +131,7 @@ func (pod *podInfo) delContainers(id string) []containerInfo {
 }
 
 // containerExists checks returns true if a container exists in the pod
-// it uses log to log inconsistencies in the data.
-func (pod *podInfo) containerExists(log logrus.FieldLogger, containerID string, cgIDp *CgroupID) bool {
+func (m *state) containerExists(pod *podInfo, containerID string, cgIDp *CgroupID) bool {
```

Comment:
nit: Maybe we can move this function a bit later (i.e. after line 217 in the same file) to be in the same place as other`state` methods.

Reply:
It seems that you don't use `m` in this commit (nor in the subsequent events). What is the reason for this change?

Reply:
We are using it for the log message:
```
-               log.WithFields(logrus.Fields{
+               m.log.WithFields(logrus.Fields{
+                       "pod-id":        pod.id,
```

a bit below.

Reply:
Moved the function!